lib/features/login/presentation/login_controller.dart

```diff
@@ -170,6 +178,7 @@ class LoginController extends BaseController {
   }
 
   void handleLoginPressed() {
+    _saveRecentLoginUsername();
```

Comment:
We should only save the email when it is in valid. Correct email format `x@b.com`